includes/theme_status_messages.inc

```diff
@@ -17,12 +17,12 @@
 function campaignion_foundation_status_messages($variables) {
   $output = '';
   $status = [
-    'status'  => ['heading' => t('Status message'), 'class' => 'success'],
-    'error'   => ['heading' => t('Error message'), 'class' => 'alert'],
-    'warning' => ['heading' => t('Warning message'), 'class' => 'warning'],
+    'status'  => ['heading' => t('Status message'), 'class' => 'success', 'role' => 'status'],
+    'error'   => ['heading' => t('Error message'), 'class' => 'alert', 'role' => 'alert'],
+    'warning' => ['heading' => t('Warning message'), 'class' => 'warning', 'role' => 'status'],
   ];
   foreach (drupal_get_messages($variables['display']) as $type => $messages) {
-    $output .= '<div class="callout ' . ($status[$type]['class'] ?? $type) . "\" data-closable>\n";
+    $output .= '<div class="callout ' . ($status[$type]['class'] ?? $type) . "\" data-closable role=\"" . $status[$type]['role'] . "\">\n";
```

Comment:
Agree with the choice of role for each type :+1: 

I'm afraid there is just 1 use case where the role is relevant, that’s server side validation (usually only kicks in for duplicate values in unique form fields). Clientside validation does not render status messages and status messages in $messages never update. Useful as it would be to have the role in clientside validation as well, it's probably not worth the trouble for Drupal 7 anymore.

Reply:
Please check the quotes: `"`are only needed when using special characters like `\n` or parsing variables. Here you can stick to single quotes: `'" data-closable role="'`. See [php strings](https://www.php.net/manual/en/language.types.string.php).

You could also refactor the code for better readability, for example:
```php
foreach (drupal_get_messages($variables['display']) as $type => $messages) {
  $class = $status[$type]['class'] ?? $type;
  $role = $status[$type]['role'] ?? 'status';
  $output .= "<div class=\"callout ${class}\" role=\"${role}\" data-closable>\n";
```

Reply:
Thanks!